.github/workflows/build.yml

```diff
@@ -90,13 +90,14 @@ jobs:
         run: |
           python -m pip install --upgrade pip setuptools wheel twine
           pip install -r requirements.txt
-      - name: Materialize Fixed Version
+      - name: Build Release with setuptools
+        if: github.ref=='refs/head/main'
         run: |
-          python .github/build/set_version.py > graspologic/version/version.txt
-          cat graspologic/version/version.txt
-      - name: Build with setuptools
+          python setup.py sdist
+      - name: Build Prerelease with setuptools
+        if: github.ref='refs/head/dev'
         run: |
-          python setup.py build sdist
+          python setup.py egg_info -b "dev$GITHUB_RUN_ID" sdist
```

Comment:
I may need to screw with this once we're on dev, we won't know if this works until we merge